File: testresources/java/accuracy/src/domain/direct/violating/AccessInstanceVariableConstant.java
package domain.direct.violating;

import technology.direct.dao.UserDAO;

public class AccessInstanceVariableConstant {
	private UserDAO userDao;
	public String sniName;
	public String sniMessage;
	
	public AccessInstanceVariableConstant(){
	}
	 
	public void testAccessFinalAttribute() {
		sniMessage = userDao.message;
	}
	
}

File: testresources/csharp/benchmark/Domain/Direct/Violating/AccessInstanceVariableConstant.cs

namespace Domain.Direct.Violating
{
	using Domain.Direct;	
	using Technology.Direct.Dao;

	public class AccessInstanceVariableConstant
	{
		public UserDAO userDao;
		public string sniMessage;

		public void TestAccessFinalAttribute()
		{
			sniMessage = userDao.message;
		}
	}
}
